pages/common/az-storage-account.md

```diff
@@ -0,0 +1,21 @@
+# az storage account
+
+> Manage storage accounts in Azure.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/storage/account>.
+
+- Create a account in a storage account:
```

Comment:
Wait, based on the doc this command is for create an storage account itself :thinking: 
```suggestion
- Create an storage account:
```